conda/activate.py

```diff
@@ -894,6 +924,68 @@ def _translation(match):
         return tuple(unix_path.split(":"))
 
 
+def unix_path_to_native(
```

Comment:
There might well be but I was looking to create a reflection of the `native_path_to_unix()` function immediately above to perform the reverse function.
`native_path_to_unix()` is used several times in `activate.py` but it would appear that no-one previously needed the reverse.

FWIW, in both cases, I believe the functionality is, in practice, implemented by calling the Cygwin/MSYS2 command `cygpath`, with `-u`/`-w` flags as necessary, to massage the names appropriately leaving the "do it in Python" parts of the functions unused/untested.